src/resources/views/squads/show.blade.php

```diff
@@ -217,6 +217,7 @@
     </div>
   @endcan
 
+  @if(!$squad->is_classified or $squad->getIsModeratorAttribute() or auth()->user()->isAdmin())
```

Comment:
can you use gate here ? https://github.com/eveseat/web/blob/a1348ae96b14c7e62a3a4ad5f6026383a933b8ba/src/Acl/Policies/SquadPolicy.php#L119 / https://github.com/eveseat/web/blob/a1348ae96b14c7e62a3a4ad5f6026383a933b8ba/src/WebServiceProvider.php#L413 :p

(also, adapt the gate to handle this new classified attribute)